package.json

```diff
@@ -21,22 +21,22 @@
     "apollo-upload-client": "^18.0.1",
     "axios": "^1.7.3",
     "date-fns": "^3.6.0",
-    "expo": "~51.0.37",
+    "expo": "~51.0.39",
```

Comment:
for some reason I couldn't get the builds to run locally without updating the basic expo deps. 
weird, but these are minor bumps so hopefully safe/ok.